libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/ui/SalesforceReactActivity.java

```diff
@@ -100,6 +103,9 @@ public void onDestroy() {
 
     @Override
     public void onResume() {
+        if(client != null && reactActivityDelegate != null){
+            reactActivityDelegate.onReadyCreate();
```

Comment:
if they override shouldAuthenticate() to return false, then it means they expect to run the react native code without being authenticated (they might choose to authenticate later by using the oauth bridge). In that case, we should do the onReadyCreate() without having a client.